src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/model/SwitchMeters.groovy

```diff
@@ -25,7 +31,23 @@ class SwitchMeters {
                 .findAll {it.getSwitchId() == switchId}
     }
 
+    List<MeterEntry> getMeters() {
+        northboundService.getAllMeters(switchId).meterEntries
+    }
+
     void delete(FlowMeter flowMeter) {
-        northboundService.deleteMeter(switchId, flowMeter.getMeterId().getValue())
+        delete(flowMeter.getMeterId().getValue())
+    }
+
+    DeleteMeterResult delete(Long meterId) {
```

Comment:
I think you have a mix of `SwitchMeter` and `SwitchMeters` class methods. 
Single meter should have methods like `delete(), add(), isDefault()`, and all the meters can be retrieved from switch (in this particular case I recommend not to waste time on creating `SwitchMeters` class, as the only related `getMeters()` can be moved to `SwitchExtended` class.